courses/free-courses-vi.md

```diff
@@ -141,6 +141,7 @@
 * [HTML](https://www.codehub.com.vn/HTML) - Codehub
 * [HTML Cơ Bản](https://www.codehub.com.vn/HTML-Co-Ban) - Codehub
 * [HTML Cơ Bản](https://www.youtube.com/playlist?list=PLl4nkmb3a8w135_M4YRPzYD9_6tERz3ce) - Thạch Phạm
+* [HTML Dành Cho Lập Trình Viên Web](https://zendvn.com/mien-phi-html-danh-cho-lap-trinh-vien-web-68) - ZendVN
```

Comment:
looks to me like there is an instructor name. If so: `  - Great Instructorname (ZendVN)`